Source/Extensions/Blazorise.TreeView/Internal/_TreeViewNode.razor.cs

```diff
@@ -15,6 +15,21 @@ public partial class _TreeViewNode<TNode> : BaseComponent
 {
     #region Methods
 
+    protected override async Task OnInitializedAsync()
+    {
+        foreach ( var nodeState in NodeStates ?? Enumerable.Empty<TreeViewNodeState<TNode>>() )
+        {
+            if ( nodeState.HasChildren && AutoExpandAll && !nodeState.Expanded && !nodeState.AutoExpanded )
+            {
+                nodeState.AutoExpanded = true;
+
+                await ToggleNode( nodeState );
+            }
+        }
```

Comment:
Could possible wrap in a method like `AutoExpandNodes` or something just so it's a bit more descriptive of what's it doing, Otherwise I guess this is indeed better.